src/components/emby-tabs/emby-tabs.css

```diff
@@ -13,11 +13,11 @@
     vertical-align: middle;
     flex-shrink: 0;
     margin: 0;
-    padding: 1.5em;
+    padding: 1.5em 1.5em;
     position: relative;
     height: auto;
     min-width: initial;
-    line-height: initial;
+    line-height: 1.25;
```

Comment:
Btw (somewhat unrelated to this PR and nitpicking) I think this property can be completely removed, it prevents the text from centering in the vertical axis in smaller screens